client/src/pages/TreeMap/index.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { useCallback, useState } from 'react';
+import PropTypes from 'prop-types';
+import { withGetScreen } from 'react-getscreen';
+import { Map } from '../../components/Map/map';
+import { Layer, Popup, Source, useMap } from 'react-map-gl';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import { Link } from 'react-router-dom';
+import { Button } from '../../shared';
+import { timeToObjectionText } from '../Tree/utils';
+
+
+const polygonStyle = {
+	id: 'geojson',
+	type: 'fill',
+	paint: {
+		'fill-color': '#e56666',
+		'fill-outline-color': '#000000',
+		'fill-opacity': 0.5
+	},
+};
+
+const symbolStyle = ({ image, size }) => ({
+	type:'symbol',
+	layout:{
+		'icon-image': image,
+		'icon-size':size,
+		'icon-allow-overlap': true,
+	}
+});
+
+const TreeMap = ({ isMobile, geojson }) => {
+	const [locationInfo, setLocationInfo] = useState(null);
+
+	const onClick = useCallback(event => {
+		const location = event.features && event.features[0];
+		if (!location) {return;}
+
+		setLocationInfo({
+			longitude: event.lngLat.lng,
+			latitude: event.lngLat.lat,
+			properties: location && location.properties
+		});
+	}, []);
+
+	const selectedLocationProps = (locationInfo && locationInfo.properties) || '';
+
+	const timeToObjection = timeToObjectionText(selectedLocationProps.last_date_to_objection);
+	const place = selectedLocationProps.place;
+
+	if (isMobile()) {return null;}
```

Comment:
should remain null IMO. if we don't know what to do here, at least put an image / message to the user that this feature is available only in desktop.